FastApi/app/routers/auth.py

```diff
@@ -0,0 +1,29 @@
+from fastapi import APIRouter, Depends, status, HTTPException, Response
+from fastapi.security.oauth2 import OAuth2PasswordRequestForm
+from sqlalchemy.orm import Session
+
+from .. import database, schemas, models, utils, oauth2
+
+router = APIRouter(tags=['Autentication'])
```

Comment:
Why didn't you define prefix for the API router, same as you did for posts route?

Reply:
my mistake, this is ok, I mixed up